x/mongo/driver/topology/pool_test.go

```diff
@@ -773,6 +775,45 @@ func TestPool(t *testing.T) {
 			assert.Equalf(t, 2, p.totalConnectionCount(), "pool should have 2 total connection")
 			assert.Equalf(t, 0, p.availableConnectionCount(), "pool should have 0 idle connection")
 
+			p.close(context.Background())
+		})
+		t.Run("cancel wait queue", func(t *testing.T) {
```

Comment:
```suggestion
		t.Run("canceled context in wait queue", func(t *testing.T) {
```
Could we use a more descriptive test name?

Reply:
[ef458c5](https://github.com/mongodb/mongo-go-driver/pull/1058/commits/ef458c51dfccd3237a7d3454669196014aabeb1d)